frontend/src/ui/player/index.tsx

```diff
@@ -122,6 +122,9 @@ export const InlinePlayer: React.FC<PlayerProps> = ({ className, event, ...playe
             "div.loader-container": {
                 backgroundColor: isDark ? COLORS.grey3 : "inherit",
             },
+            "div.preview-container": {
+                backgroundColor: `${COLORS.grey2} !important`,
+            },
```

Comment:
Hm on which video did you notice this? I mean I saw David showing this with the tram video but somehow I can't reproduce (trying on the current test deployment (not of this PR ofc, I mean the regular deployment) and also locally without these lines).

Reply:
You have to make your viewport/screen very narrow and then look at a video page of a portrait video (without clicking play).